src/js/logger.js

```diff
@@ -28,17 +29,23 @@ RiseVision.Common.LoggerUtils = (function() {
 
       json.company_id = companyId;
       json.display_id = displayId;
-      
-      if(location.href) {
-        var parent = getParameterByName("parent", location.href); // legacy presentations
-        var type = getParameterByName("type", location.href); // templates
-        var presentation_type = type ? type : (parent ? getParameterByName("type", parent) : "");
-        var schedule_id = parent ? getParameterByName("id", parent) : "";
-        if (presentation_type ==="sharedschedule") {
-          json.presentation_type = presentation_type;
-          json.schedule_id = schedule_id;
-          json.unique_id = getUniqueId();
-        }
+
+      var parent = utils.getQueryParameter("parent"); // legacy presentations
+      var type = utils.getQueryParameter("type"); // templates
+      var env = utils.getQueryParameter("env"); // endpoint
+      var viewerId = utils.getQueryParameter("viewerId");
+
+      var presentation_type = type ? type : (parent ? utils.getQueryStringParameter("type", parent) : "");
+      var endpoint_type = env ? env : (parent ? utils.getQueryStringParameter("env", parent) : "");
+      var viewer_id = viewerId ? viewerId : (parent ? utils.getQueryStringParameter("viewerId", parent) : "");
```

Comment:
@stulees I'm not sure this is needed. Viewer is not going to have `viewerId` URL param. We can leave it as is for the same of consistency. It won't hurt.

Reply:
@olegrise We spoke, the `viewerId` will be needed as a URL param, so this code should remain intact. Cheers